common/web/lm-worker/build-wrapper.js

```diff
@@ -0,0 +1,108 @@
+import fs from 'fs';
+
+import convertSourcemap from 'convert-source-map'; // Transforms sourcemaps among various common formats.
+                                                   // Base64, stringified-JSON, end-of-file comment...
+
+let INCLUDE_SRCMAPS = false;
+
+let sourceFromArgs;
+let destFromArgs;
+
+function doHelp(errCode) {
+  console.log(`
+Summary:
+  Creates a "wrapped" version of the lm-worker for compilation into and inclusion as part
+  of a different JS bundle for later use as the core of a predictive-text WebWorker.
+
+Usage:
+  node build-wrapper.js <input-file> [options...]
+
+Parameters:
+  <input-file>:  Fully-bundled and compiled JS file to be wrapped.
+
+Options:
+  --help            Shows this script's documentation
+  --out=<out-file>  Specifies the destination path for the wrapped output.
+
+                    If missing, the output will be placed next to the source and given
+                    the same path, but with '.wrapped.js' replacing the original '.js'
+                    extension.
+  --sourceMap       Includes the script's original sourcemaps within the wrapped output
+` );
+  process.exit(errCode || 0);
+}
+
+if(process.argv.length > 2) {
+  for(let i = 2; i < process.argv.length; i++) {
+    const arg = process.argv[i];
+
+    switch(arg) {
+      case '--help':
+        doHelp();
+        break;
+      case '--sourceMap':  // bc TS uses this exact flag.  esbuild... uses sourcemap (in the JS config)
```

Comment:
```suggestion
      case '--sourceMap':  // bc TS uses this exact flag.  esbuild... uses sourcemap (in the JS config)
      case '--sourcemap':
      case '--source-map':
```

Completely optional though ;-)